### DISCUSSION TITLE
[mini.pairs] Disable pairs for searching

### DISCUSSION THREAD

**Discussion by pkropachev** [nvim-mini/mini.nvim#1083] [Q&A] [answered]
Hello!

Could you please advice if there is a way to disable pairs for searching. For example, I search pattern `test(`. I'd like to avoid pairing in this case.

**Comment by echasnovski** [comment, marked as answer]
With default config it should not do that, as [pairs are not created for Command mode](https://github.com/echasnovski/mini.nvim/blob/657ef75b52a548090fb7906735517b7ddbf39e88/doc/mini-pairs.txt#L118).

Here is what I can suggest here:
- Disable creating pairs in Command mode. This will stop adding pair for any activity inside Command line: entering command, search pattern, user input, expression to evaluate after `<C-r>=`, etc. There is no more granular pair creation: they all can be simultaneously either present or not.
- Deliberately press `<C-v>` prior to the character you don't want to lead to the whole pair. I personally use this.

**Reply by pkropachev** [reply]
Thanks!
